packages/forklift-console-plugin/locales/en/plugin__forklift-console-plugin.json

```diff
@@ -410,9 +423,11 @@
   "URL of the providers API endpoint. The URL must be a valid endpoint for the provider type, see\n      the documentation for each provider type to learn more about the URL format.": "URL of the providers API endpoint. The URL must be a valid endpoint for the provider type, see\n      the documentation for each provider type to learn more about the URL format.",
   "User ID": "User ID",
   "Username": "Username",
+  "Validate Certificate": "Validate Certificate",
```

Comment:
I don't see this string in the screenshot - when it is presented?

Reply:
it's `aria-label` for the checkbox (for accessibility)

Reply:
actually it can be removed - the checkbox already has a proper label